products/BellHybrid/apps/application-bell-meditation-timer/windows/MeditationCountdownWindow.cpp

```diff
@@ -79,6 +79,19 @@ namespace gui
         timer->setMargins(gui::Margins(0, countdownStyle::timer::marginTop, 0, 0));
         timer->setAlignment(Alignment(Alignment::Horizontal::Center, Alignment::Vertical::Center));
 
+        bottomDescription = new gui::TextFixedSize(
```

Comment:
```suggestion
        bottomDescription = new TextFixedSize(
```

It's already in `namespace gui`, so it's not needed here :wink:

Reply:
done